src/classes/LookupConfigController.cls

```diff
@@ -1,5 +1,35 @@
-public with sharing class LookupConfigController {
+public class LookupConfigController {
+	public String objectName { get; set; }
```

Comment:
We'll probably end up replacing the variables on lines 2-7 with a single variable for the new custom object that will house the fields - but for now we can keep these lines.  No need to change anything, but i just wanted to note it for future reference.  👍